packages/image_picker/ios/Classes/ImagePickerPlugin.m

```diff
@@ -93,6 +93,11 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     _arguments = call.arguments;
 
     int imageSource = [[_arguments objectForKey:@"source"] intValue];
+    if ([_arguments objectForKey:@"maxDuration"]) {
+        double max = [[_arguments objectForKey:@"maxDuration"] doubleValue];
```

Comment:
nits: NSTimeInterval maxDuration

Reply:
Also if maxDuration is NSNull from dart, this `max` here will be come 0.

Reply:
Could you explain a case where max would become 0 more clearly, please? Because if NSNull is passed from dart, the if shouldn't pass.

Reply:
As far as I remember, 
`if ([_arguments objectForKey:@"maxDuration"])`  will return true if [_arguments objectForKey:@"maxDuration"] is NSNull.
You could also change it to 
`if ([[_arguments objectForKey:@"maxDuration"] isKindOf:[NSNumber class]])`